.github/workflows/ios_deploy_staging_to_firebase.yml

```diff
@@ -54,6 +54,7 @@ jobs:
           ENV: ${{ secrets.ENV }}
         run: |
           echo -e "$ENV" > .env.staging
+          echo "RELEASE_NOTE_CONTENT="$((git log -1 --merges | grep "\[") | grep . && echo "" || echo $(git log -1 --merges --format=%B))""
```

Comment:
Not sure whether this `RELEASE_NOTE_CONTENT` env applies for the `wzieba/Firebase-Distribution-Github-Action` action or [@fastlane.firebase_app_distribution](https://firebase.google.com/docs/app-distribution/ios/distribute-fastlane) plugin. I don't see the usage of 
- https://github.com/wzieba/Firebase-Distribution-Github-Action?tab=readme-ov-file#releasenotes 
   https://github.com/nimblehq/flutter-templates/blob/2d294909469712a170a80b96774171f76ebf79ea/.github/workflows/android_deploy_production.yml#L55-L61


- and [@fastlane.firebase_app_distribution's `release_notes`](https://firebase.google.com/docs/app-distribution/ios/distribute-fastlane#distribute) 
   https://github.com/nimblehq/flutter-templates/blob/2d294909469712a170a80b96774171f76ebf79ea/bricks/template/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D/ios/fastlane/Fastfile#L214